health/warnings.go

```diff
@@ -18,17 +21,37 @@ var updateAvailableWarnable = Register(&Warnable{
 	Title:    "Update available",
 	Severity: SeverityLow,
 	Text: func(args Args) string {
-		return fmt.Sprintf("An update from version %s to %s is available. Run `tailscale update` or `tailscale set --auto-update` to update.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		if version.IsMacAppStore() || version.IsAppleTV() {
+			return fmt.Sprintf("An update from version %s to %s is available. Open the App Store to update now.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else if version.IsMacSys() {
+			return fmt.Sprintf("An update from version %s to %s is available. Update now from the Tailscale settings.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else if version.IsWindowsGUI() {
+			return fmt.Sprintf("An update from version %s to %s is available. Update now from the Tailscale menu.", args[ArgCurrentVersion], args[ArgAvailableVersion])
+		} else if runtime.GOOS == "android" {
+			return fmt.Sprintf("An update from version %s to %s is available. Open the Play Store to update now.", args[ArgCurrentVersion], args[ArgAvailableVersion])
```

Comment:
Might not be Play Store if sideloaded?

Reply:
I tried looking for something like this, but we don't have a way of detecting the APK source from Go at the moment AFAICT. Suggesting people to use the Play Store is still better than... telling them to run a CLI command on a phone?

Reply:
Dropped the _how_ on all non-CLI platforms.